main/src/cgeo/geocaching/models/Geocache.java

```diff
@@ -1510,10 +1510,14 @@ public Waypoint getWaypointById(final int id) {
      * Detect coordinates in the personal note and convert them to user defined waypoints. Works by rule of thumb.
      */
     public boolean parseWaypointsFromNote() {
+        return parseWaypointsFromText(getPersonalNote(), false);
+    }
+
+    public boolean parseWaypointsFromText(final String text, final boolean updateDb) {
```

Comment:
Since we take care of a `null` entry, I'd use `@Nullable` here as well for `text`.